doc/HOWTO_reservations.md

```diff
@@ -5,12 +5,24 @@ Instruments have a time schedule where they can be reserved. A Reservation
 generally refer to a user scheduling the use of the instrument but also can
 refer to the instrument being offline or reserved for an administrative task.
 
-Reservations on instruments are a kind of product the user purchase and can
-have different ways of charging depending on the Instrument's pricing mode. So
-whenever a new reservation is created an order is created or updated to include
-it as part of its items.
+Instruments are a type of Product which whose usage can be purchased by users
+through Reservations. So whenever a new reservation is created, an order is
+created or updated to include the Instrument as part of its items with a cost
```

Comment:
`item` is a type of product 😅 

Maybe we can say _So whenever a new reservation is created, an order is created or updated to include the Instrument as part of its ~items~ *order details*_?

Reply:
makes sense